apps/rehabstod/src/components/SickLeave/SickLeaveDegreeInfo.tsx

```diff
@@ -3,15 +3,21 @@ import { IDSIcon } from '@frontend/ids-react-ts'
 export function SickLeaveDegreeInfo({ degrees }: { degrees: number[] }) {
   const hasSeveralDegrees = degrees.length > 1
 
+  if (degrees.length === 0) {
+    return <td>Okänt</td>
```

Comment:
This converts the component to a table cell, perhaps we could just use `<span>Okänt</span>` instead.